cli/schemas/interchain-tokenlist.ts

```diff
@@ -13,6 +13,10 @@ export const remoteToken = z.object({
 export const interchainToken = z.object({
   tokenId: hash,
   tokenAddress: address,
+  tokenManagerAddress: address,
+  deployerAddress: address,
```

Comment:
can also add salt since it's needed for deployment. any other info that the portal stores?

Reply:
we also have:
* "kind" (canonical, interchain, or custom)
* original tx hash of deployment on the src chain
that's it, i don't think we'll need these though in these configs

Reply:
cool, definitely should add the kind / `tokenType`. I'd add deployment tx hash as well for convenience. we don't need to publish tx hash in the main configs for now but good to have here.